Neos.Fusion/Classes/View/FusionView.php

```diff
@@ -54,10 +56,8 @@ class FusionView extends AbstractView
 
     /**
      * The parsed Fusion array in its internal representation
-     *
-     * @var array
      */
-    protected $parsedFusion;
+    protected FusionConfiguration $parsedFusion;
```

Comment:
Changing this from array to FusionConfiguration is also breaky.

Reply:
Yes, but we evaluated that to be okayish - see https://github.com/neos/neos-development-collection/pull/4425#discussion_r1319460251 for a workaround ;)

Reply:
agreed that was solved